test/exit/capture_callback.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var Raven = require('../../');
+var assert = require('assert');
+var dsn = 'https://public:private@app.getsentry.com/269';
+
+var nock = require('nock');
+var scope = nock('https://app.getsentry.com')
+  .filteringRequestBody(/.*/, '*')
+  .post('/api/269/store/', '*')
+  .reply(200, 'OK');
+
+Raven.config(dsn).install(function (err, sendErr, eventId) {
+  scope.done();
+  assert.equal(sendErr, null);
+  assert.ok(err instanceof Error);
+  assert.equal(err.message, 'derp');
+  console.log('exit test assertions complete');
+  process.exit(20);
+});
+
```

Comment:
I don't see any need for it, what's the case you're thinking it would help check against? We already check for `scope.done()` at the beginning of the onFatalError callback, and the mocha test case checks that this test exits with code 20, so the only exit path that will pass there has to go through the `scope.done()` here.